### PR TITLE
Closes #71 Migrated to Swift 3, updated to Alamofire 4.0

### LINKED ISSUE #71: Update repo for iOS 10 SDK
Repository should be migrated and updated to build with iOS 10 SDK before the official release.

### PR DESCRIPTION
**Pull request by nikola-mladenovic**
@cneijenhuis - in order to use Swift 3 accross our project, we had to start using the latest Alamofire version (4.0). The codebase has been updated accordingly, tests as well.

After merging this one, I'd like to go ahead and quickly refactor our `CTError` to be aligned to `AFError` from the latest version. If you agree, I'll create a separate issue for that?

### CODE REVIEW

File: Source/AuthManager.swift [hunk 1]
```diff
@@ -78,25 +78,25 @@ public class AuthManager {
 
     /// The URL used for requesting token for client credentials and refresh token flow.
     private var clientCredentialsUrl: String? {
-        if let config = Config.currentConfig, baseAuthUrl = config.authUrl where config.validate() {
+        if let config = Config.currentConfig, let baseAuthUrl = config.authUrl , config.validate() {
```
Comment (cneijenhuis) [inline Source/AuthManager.swift]: `config.authUrl ,` is the whitespace here intentional?

Comment (cneijenhuis) [inline Source/AuthManager.swift]: After reviewing the PR, it seems that sometimes there is a whitespace before the `,` and sometimes not. I suppose the XCode replaced the `where`?

Comment (nikola-mladenovic) [inline Source/AuthManager.swift]: That is correct. I'm going to clean up formatting where necessary before merging this one.

**Comment by nikola-mladenovic** [comment]
Also, the tests are passing, but there seems to be an issue with the new Xcode 8 travis image, and CocoaPods. Will look into it as well.

File: Source/AuthManager.swift (hunk 1 above)
Comment (cneijenhuis) [inline Source/AuthManager.swift]: 👍

**Comment by nikola-mladenovic** [comment]
Fixed the travis issue with CocoaPods and updated formatting across source + test files. Will proceed with merge after travis checks complete.

**Pull request merged**